lib/evss/response.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+require 'common/client/concerns/service_status'
+require 'common/models/base'
+
+module EVSS
+  class Response < Common::Base
+    include Common::Client::ServiceStatus
+
+    attribute :status, Integer
+
+    def initialize(status, response = nil)
+      self.status = status
+      super(response.body) if response
+    end
+
+    def ok?
```

Comment:
Is this used anymore?  I'm fine with leaving it but it looks like you've removed all uses of it.